src/pages/Books/Books.js

```diff
@@ -1,7 +1,132 @@
-import classes from "./Books.module.scss";
+import {useEffect, useState} from "react";
+import {FormControl, InputLabel, Pagination, Select, MenuItem, OutlinedInput, Checkbox, Button} from "@mui/material";
+import Box from "@mui/material/Box";
+import ListItemText from "@mui/material/ListItemText";
+import {getBooks} from "api/api";
+import Book from "components/Book";
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+    PaperProps: {
+        style: {
+            maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+            width: 250,
+        },
+    },
+};
+
+const names = [
+    'drama',
+    'crime',
+    'thriller',
+    'romance',
+    'history',
+    'fiction'
+];
+
 
 const Books = () => {
-    return <h1>Books page!</h1>
+    const booksPerPage = 5;
+    const [page, setPage] = useState(1);
+    const [sort, setSort] = useState('DEFAULT');
+    const [genres, setGenres] = useState([]);
+    const [books, setBooks] = useState([]);
+    const [totalPages, setTotalPages] = useState(0);
+    useEffect(()=>{
+        getBooks(page, genres, sort).then(res=>{
+            setBooks([...res.books]);
```

Comment:
you don't need to spread this value (same for all 3 uses)
since we are not ever directly changing react state variables we can pass by reference here.